src/templates/finops-hub/modules/storage.bicep

```
@@ -76,6 +79,10 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' = {
   tags: union(tags, contains(tagsByResource, 'Microsoft.Storage/storageAccounts') ? tagsByResource['Microsoft.Storage/storageAccounts'] : {})
   properties: {
     supportsHttpsTrafficOnly: true
+    encryption: {
+      keySource: 'Microsoft.Storage'
+      requireInfrastructureEncryption: true
```

Should the parameter be used here? Do we need to make the `encryption` property null if this is not enabled?
```suggestion
      requireInfrastructureEncryption: enableInfrastructureEncryption
```